packages/dds/sequence/src/defaultMap.ts

```diff
@@ -21,13 +21,16 @@ import {
 	// eslint-disable-next-line import/no-deprecated
 	IValueOpEmitter,
 	IValueType,
-	IValueTypeOperationValue,
 	ISharedDefaultMapEvents,
 	IMapMessageLocalMetadata,
 	SequenceOptions,
+	IValueTypeOperationValue,
 } from "./defaultMapInterfaces";
-import { IntervalOpType, SerializedIntervalDelta } from "./intervals";
+import { SerializedIntervalDelta, IntervalDeltaOpType } from "./intervals";
 
+function isMapOperation(op: unknown): op is IMapOperation {
+	return typeof op === "object" && op !== null && "type" in op && op.type === "act";
```

Comment:
op!== null is redundant cause we already checked for typeof op = "object". Also, I think you can just check op.type. 
```suggestion
	return typeof op === "object" && op.type === "act";
``` 
maybe even just the last condition alone works.

Reply:
Tony and I tried this when we went through and refactored the types - I get errors if I don't include the null check (as well as the others).